ext/drbbdup/drbbdup.h

```diff
@@ -257,8 +258,32 @@ typedef struct {
      * thread before it becomes a candidate for dynamic generation.
      */
     ushort hit_threshold;
+    /**
+     * Determines whether drbbdup should track a variety of statistics. Note keeping track
+     * of statistics incurs additional overhead and it is not recommended at deployment.
+     *
+     * Set to true if the client calls drbbdup_get_stats().
+     */
+    bool is_stat_enabled;
 } drbbdup_options_t;
 
+/**
+ * Various statistics related to drbbdup.
+ */
+typedef struct {
+    /** Number of fragments which have case handling turned off. */
+    unsigned long no_dup_cnt;
```

Comment:
suggest: just spell out "count" for clarity, here and below

Reply:
Done.